README.md

```diff
@@ -258,3 +258,19 @@ You also can use the provided `Dockerfile` to build your own image:
 ```shell
 docker build -t sgpt .
 ```
+
+## Development
+Install dependencies:
```

Comment:
I found `pip install -r requirements.txt` to be a much simpler way of installing the requirements.

Reply:
Thanks, appreciate the feedback.